src/main/resources/application.yml

```
@@ -0,0 +1,38 @@
+  server:
+    base-url: http://localhost:8000 # AI 서버 개발자(현준님)에게 받은 실제 주소로 변경해야 합니다.
+
+cloud:
+  aws:
+    credentials:
+      access-key: ${s3-access-key}
+      secret-key: ${s3-secret-key}
+    s3:
+      bucket: ${s3-bucket-name}
+    region:
+      static: ap-northeast-2
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)